ted/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -143,6 +143,10 @@ public class DrivetrainSubsystem extends SubsystemBase {
   private SwerveDrivePoseEstimator swervePoseEstimator = null;
   private Pose2d currentPosition = new Pose2d();
   private ArrayDeque<Pose2d> historicPositions = new ArrayDeque<Pose2d>(PositionHistoryStorageSize + 1);
+  // Standard deviations for poseEstimator updates
+  // The wpilib matrix constructor requires sizes specified as Nat types. 
+  private Matrix<N3,N1> visionStdDev = MatBuilder.fill(Nat.N3(), Nat.N1(), new double[]{0.7, 0.7, 10});
+  private Matrix<N3,N1> odometryStdDev = MatBuilder.fill(Nat.N3(), Nat.N1(), new double[]{0.1, 0.1, 0.1});
```

Comment:
Add comment that these are the default recommended settings.

Reply:
added